README.md

```diff
@@ -1,40 +1,46 @@
 
 ## Usage
 
-### Windows
-* [Downlaod the zip file](https://github.com/aryan9600/cowin-notifier/releases/download/v0.1/cowin-notifier-windows.zip) and unzip it.
+### 💻 Windows
+* [Download the zip file](https://github.com/aryan9600/cowin-notifier/releases/download/v0.1/cowin-notifier-windows.zip) and unzip it.
 * Open the folder containing the file, do a `Shift + Right Click` and select "Open Powershell here."
-* Run `./cowin-notifier "state_name", "district_name", "age_group"`
-![Alt text](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/WhatsApp%20Image%202021-05-06%20at%2021.23.49.jpeg)
+* Run `./cowin-notifier "state_name" "district_name" "age_group"`
+![Windows Screenshot](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/WhatsApp%20Image%202021-05-06%20at%2021.23.49.jpeg)
 * Minimize the Powershell window.
 * If a slot opens up for your age group and district, you'll recieve a notification. __Make sure that you don't close the Powershell window.__
 
-### macOS
+### 🍎 macOS
 * [Download the zip file](https://github.com/aryan9600/cowin-notifier/releases/download/v0.1/cowin-notifier-macos.zip) and unzip it.
-* Open your terminal and `cd` into your default Downloads directory. macOS users can fire up Spotlight(cmd+Space), search for Terminal and open it.
-
-![Alt text](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/Screen%20Shot%202021-05-05%20at%2015.59.44.png)
-
-* Run `cd ~/Downloads`
-
-* Run `./cowin-notifier "state_name", "district_name", "age_group"`
-![Alt text](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/Screen%20Shot%202021-05-05%20at%2016.06.28.png)
-
+* Open your terminal and `cd` into your default Downloads directory where you have downloaded the file. macOS users can fire up Spotlight using `cmd + Space`, search for Terminal and open it.
+![macOS Screenshot-1](https://github.com/mintbomb27/cowin-notifier/raw/main/assets/macOS%20spotlight.png)
+* Run `cd ~/Downloads` 
+* Run `chmod +x cowin-notifier`
+* Run `./cowin-notifier "state_name" "district_name" "age_group"`
+![macOS Screenshot-2](https://github.com/mintbomb27/cowin-notifier/raw/main/assets/macOS%20notif.png)
 * Minimize your terminal.
 * If a slot opens up for your age group and district, you'll recieve a notification. __Make sure that you don't close the Terminal.__
 
-![Alt text](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/Screen%20Shot%202021-05-05%20at%2017.46.10.png)
+![macOS Screenshot-3](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/Screen%20Shot%202021-05-05%20at%2017.46.10.png)
 
-### Linux
+### 🐧 Linux
 * We use the [reqwest](https://github.com/seanmonstar/reqwest) crate, which has some specific [requirements for Linux](https://github.com/seanmonstar/reqwest#requirements). Please follow them first.
```

Comment:
Could you also please rephrase this asking the user to install the requirements and dependencies instead of  `Please follow them first`